app/controllers/about/supporting_organisations_controller.rb

```diff
@@ -0,0 +1,11 @@
+class About::SupportingOrganisationsController < ApplicationController
+  skip_before_action :authenticate_user!
+
+  def index
+    @supporting_orgs = SupportingOrganisation.all.select(:name, :slug)
```

Comment:
```suggestion
    @supporting_orgs = SupportingOrganisation.select(:name, :slug)
```

You shouldn't need the `all`

Reply:
You probably also want to include their logo as part of this to avoid an `n+1`